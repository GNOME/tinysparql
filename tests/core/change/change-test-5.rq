select ?u { ?u <http://tracker.api.gnome.org/ontology/v3/fts#match> 'resource' } order by ?u
