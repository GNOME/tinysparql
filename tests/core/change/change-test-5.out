"http://example/fts1"
"http://example/fts3"
"http://example/fts4"
